srml/executive/src/lib.rs

```diff
@@ -435,12 +429,17 @@ mod tests {
 		}
 	}
 
-	type SignedExtra = (system::CheckNonce<Runtime>, balances::TakeFees<Runtime>);
+	type SignedExtra = node_runtime::SignedExtra<Runtime>;
```

Comment:
this shouldn't be pulling in random types/configuration from `node_runtime`. it should formulate its own `SignedExtra` type as needed, just as it formulates its own config trait implementations.